model/factory.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+import tensorflow as tf
+import ruamel.yaml
+
+from model.models import ForwardTransformer, Aligner
+
+
+def tts_ljspeech(version='v1') -> [tf.keras.models.Model, dict]:
+    config_name = f'ljspeech_tts_config_{version}.yaml'
+    weights_name = f'ljspeech_tts_weights_{version}.hdf5'
+    remote_dir = 'https://public-asai-dl-models.s3.eu-central-1.amazonaws.com/TransformerTTS/api_weights/'
+    config_path = tf.keras.utils.get_file(config_name, remote_dir + config_name)
+    weights_path = tf.keras.utils.get_file(weights_name, remote_dir + weights_name)
+    return tts_custom(config_path, weights_path)
+
+
+def tts_custom(config_path: str, weights_path: str) -> [tf.keras.models.Model, dict]:
```

Comment:
I am pretty sure that type highlighting won't work with this (instead use Tuple[tf.keras.models.Model, dict].

Reply:
True.
fiy from python 3.9:
https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections